react/react.py

```diff
@@ -96,7 +98,7 @@ def to_json(data):
         :return: Received data in json format
         :rtype: str
         """
-        if isinstance(data, str):
+        if isinstance(data, six.string_types):
```

Comment:
I think it's safer to check for either type of text input. If a bytestring is passed in here, that shouldn't get passed to `json.dumps`. Do you disagree?